src/op/nms.h

```diff
@@ -82,11 +80,13 @@ class nms
         int32_t add_ipv6_cidr(const char *a_buf, uint32_t a_buf_len);
         int32_t contains_ipv4(bool &ao_match, const char *a_buf, uint32_t a_buf_len);
         int32_t contains_ipv6(bool &ao_match, const char *a_buf, uint32_t a_buf_len);
+        int32_t contains_ipv4_prefix(bool &ao_match, const char *a_buf, uint32_t a_buf_len);
```

Comment:
Do not see any definition for this cc. I think this could be removed too if not used